packages/nuxt-module/package.json

```diff
@@ -0,0 +1,73 @@
+{
+  "name": "@netlify/nuxt",
+  "version": "0.0.0",
+  "description": "Nuxt module providing local emulation of the Netlify environment",
+  "type": "module",
+  "engines": {
+    "node": "^20.6.1 || >=22"
+  },
+  "main": "./dist/module.mjs",
+  "typesVersions": {
+    "*": {
+      ".": [
+        "./dist/types.d.mts"
+      ]
+    }
+  },
+  "exports": {
+    ".": {
+      "types": "./dist/types.d.mts",
+      "import": "./dist/module.mjs"
+    }
+  },
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "build": "nuxt-module-build prepare && nuxt-module-build build",
+    "dev": "npm run dev:prepare && nuxi dev playground",
+    "dev:build": "nuxi build playground",
+    "dev:prepare": "nuxt-module-build build --stub && nuxt-module-build prepare && nuxi prepare playground",
+    "lint": "eslint .",
+    "prepack": "npm run build",
+    "publint": "npx -y publint --strict",
+    "test": "vitest run",
+    "//": "Workaround to run this package's eslint in CI",
+    "test:ci": "npm run lint && npm run test:types && npm run test",
```

Comment:
the current CI workflows make very specific assumptions about all packages, so I just squeezed this into `test:ci`